### DISCUSSION TITLE
Howdy

### DISCUSSION THREAD

**Discussion by DangarStu** [timmbogner/Farm-Data-Relay-System#235] [General]
Hi all

Just wanted to say thanks to Timm for creating FDRS and introduce myself. I live in Australia and I'm looking at using FDRS for boat monitoring and oyster farming (remote monitoring and control of water pumps etc) 

I work as a professional software developer and tech writer so I'm hoping to be able to contribute to the project as I go, but I'm sure there will be lots of questions before we get to that stage!

I now have the UART Gateway installed on a Sailor Hat ESP32 with external Ra-01 and OLED. It's a very cool little board that runs off a car battery with NMEA support. https://docs.hatlabs.fi/sh-esp32/

![IMG_1167](https://github.com/user-attachments/assets/3104d541-6ec8-40c9-bedd-f2e6daee26e7)

I'm also very interested in controlling linear actuators over a slow network like LoRa.

Nice to meet you all,
Stu

**Comment by timmbogner** [comment]
Hey Stu,
That sounds awesome! I remember seeing the SailorHat project a long time ago and had almost forgotten about it. Glad you reminded me, I now have one on the way!
It's been a chaotic year for me and I haven't put in nearly as much time to improving this project as I'd like. I do think that the system is in a reasonably stable state, however. If you find anything that's out of order, please don't hesitate to let me know. 
Thanks for the intro! Definitely feel free to ask any questions!
Timm

**Reply by DangarStu** [reply]
Thanks for your reply Timm. I'm tempted to talk to the guys over at Hat Labs as they mention on their website that they do custom boards to meet customer requirements. It would be interesting to see a board just like the one they have only with a LoRa module and solar charger in place on the NMEA module. It isn't hard to add one to the protoboard area but I think it would be a great custom board to pair with FDRS. I used to run a boating related YouTube channel that I'm in the process of rebooting and I think this remote monitoring would be a great first set of videos.

Stu
